tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```
@@ -0,0 +1,35 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite with ICs and LBCs derived from the RRFS forecasts mapped onto
+    3-km regular grid (rrfs*.conus.grib2). This test uses old v1 sfc_data,
+    not the v2 fractional grid sfc_data.
```

```suggestion
    This test is to ensure that the workflow running in community mode
    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2 for
    ICs and LBCs. This test uses old v1 sfc_data, not the v2 fractional grid sfc_data.
```

Maybe just breaking up the sentence makes it clearer that it refers to the RRFS CONUS forecasts. It's also unclear what "This test uses old v1 sfc_data, not the v2 fractional grid sfc_data" means. What impact does this have? What is the v2 fractional grid sfc_data, and what is the switch that enables this change?